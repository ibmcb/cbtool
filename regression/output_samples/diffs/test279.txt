Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.254,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.0.254,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_41,none --operation=vm-capture --cn=TESTCLOUD --uuid=FD4D2B17-AB08-53BD-979C-E273A0006F5A --daemon". The process id is 11684.
|ai_1      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |69.139.182.253  |1374526384     |none                                    |F4BAAE9B-7589-5CA4-8980-DCBDCA4102B2
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |83.212.20.114   |1374526402     |none                                    |3E0E082E-3CC9-58D2-96AF-2F6559324443
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |218.3.95.25     |1374526442     |none                                    |4FD4E473-B901-535E-81B0-B1D8E6D698DB
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |86.43.50.187    |1374526461     |none                                    |F8B6C5F5-69E7-54E3-ABA6-76C0E27513C8
|vm_26         |db2                 |copper32    |58.121.101.94   |simhostc4                  |SUT                 |ai_1    |none       |1A394EC6-639B-5FBF-8C29-32A384143BB5
|vm_28         |client_daytrader    |nano32      |69.139.182.253  |simhostd4                  |LG                  |ai_1    |none       |5003ABCD-1E1C-5800-B6F0-8436DC8FBEAE
|vm_33         |client_daytrader    |nano32      |83.212.20.114   |simhostd1                  |LG                  |ai_3    |none       |6FB95A0B-DC69-5BCA-83D4-EF6A5FDE7418
|vm_34         |was                 |iron32      |141.110.188.78  |simhostb2                  |SUT                 |ai_3    |none       |E679191D-E766-5120-B1AC-B359C5F48C44
|vm_35         |db2                 |copper32    |111.33.159.57   |simhostb4                  |SUT                 |ai_3    |none       |508E8790-2816-5B0E-B5EA-25A85B4E3E5B
|vm_36         |client_daytrader    |nano32      |218.3.95.25     |simhostd4                  |LG                  |ai_4    |none       |8262E811-541A-5386-8790-420D769BCF89
|vm_37         |was                 |iron32      |167.6.69.81     |simhostb1                  |SUT                 |ai_4    |none       |4DEF67D9-E330-5E87-8DA1-E1A5C8060BF7
|vm_38         |db2                 |copper32    |87.201.50.211   |simhostb0                  |SUT                 |ai_4    |none       |57F691F0-F925-5665-BF65-1D8523FE67EC
|vm_39         |netclient           |platinum64  |86.43.50.187    |simhosta0                  |SUT                 |ai_5    |none       |C3B97918-FB0A-5D60-B179-C521859C2641
|vm_40         |netserver           |iron32      |14.75.181.209   |simhostc2                  |SUT                 |ai_5    |none       |4CEAF1BD-1B5A-541D-B8DB-84C44699AADE
|vm_43         |was                 |iron32      |110.90.57.157   |simhostb3                  |SUT                 |ai_1    |none       |0BEDE58A-3523-51CF-966D-479F370F6227
|Used Memory                                          |40.82M                        
|Uptime (in seconds)                                  |13565298                      
|Total Connections Received                           |6494906                       
|Total Commands Processed                             |93826715                      
|Number of Keys                                       |516                           
|Storage Size                                         |16112607216.0                 
|Data Size                                            |13823086424.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.37385165                 
|Collections                                          |60                            
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|5                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|3                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|2                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|1                             
