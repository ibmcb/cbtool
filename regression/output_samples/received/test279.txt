Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.254,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.0.254,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_41,none --operation=vm-capture --cn=TESTCLOUD --uuid=FD4D2B17-AB08-53BD-979C-E273A0006F5A --daemon". The process id is 11684.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |69.139.182.253  |1374526384     |none                                    |F4BAAE9B-7589-5CA4-8980-DCBDCA4102B2
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |83.212.20.114   |1374526402     |none                                    |3E0E082E-3CC9-58D2-96AF-2F6559324443
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |218.3.95.25     |1374526442     |none                                    |4FD4E473-B901-535E-81B0-B1D8E6D698DB
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |86.43.50.187    |1374526461     |none                                    |F8B6C5F5-69E7-54E3-ABA6-76C0E27513C8

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |db2                 |copper32    |58.121.101.94   |simhostc4                  |SUT                 |ai_1    |none       |1A394EC6-639B-5FBF-8C29-32A384143BB5
|vm_28         |client_daytrader    |nano32      |69.139.182.253  |simhostd4                  |LG                  |ai_1    |none       |5003ABCD-1E1C-5800-B6F0-8436DC8FBEAE
|vm_33         |client_daytrader    |nano32      |83.212.20.114   |simhostd1                  |LG                  |ai_3    |none       |6FB95A0B-DC69-5BCA-83D4-EF6A5FDE7418
|vm_34         |was                 |iron32      |141.110.188.78  |simhostb2                  |SUT                 |ai_3    |none       |E679191D-E766-5120-B1AC-B359C5F48C44
|vm_35         |db2                 |copper32    |111.33.159.57   |simhostb4                  |SUT                 |ai_3    |none       |508E8790-2816-5B0E-B5EA-25A85B4E3E5B
|vm_36         |client_daytrader    |nano32      |218.3.95.25     |simhostd4                  |LG                  |ai_4    |none       |8262E811-541A-5386-8790-420D769BCF89
|vm_37         |was                 |iron32      |167.6.69.81     |simhostb1                  |SUT                 |ai_4    |none       |4DEF67D9-E330-5E87-8DA1-E1A5C8060BF7
|vm_38         |db2                 |copper32    |87.201.50.211   |simhostb0                  |SUT                 |ai_4    |none       |57F691F0-F925-5665-BF65-1D8523FE67EC
|vm_39         |netclient           |platinum64  |86.43.50.187    |simhosta0                  |SUT                 |ai_5    |none       |C3B97918-FB0A-5D60-B179-C521859C2641
|vm_40         |netserver           |iron32      |14.75.181.209   |simhostc2                  |SUT                 |ai_5    |none       |4CEAF1BD-1B5A-541D-B8DB-84C44699AADE
|vm_43         |was                 |iron32      |110.90.57.157   |simhostb3                  |SUT                 |ai_1    |none       |0BEDE58A-3523-51CF-966D-479F370F6227

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |40.82M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |13565298                      
|Total Connections Received                           |6494906                       
|Total Commands Processed                             |93826715                      
|Number of Keys                                       |516                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |16112607216.0                 
|Data Size                                            |13823086424.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.37385165                 
|Collections                                          |60                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |11                            
|SVMs                                                 |0                             
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |32                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |4                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |3                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |176                           

 ---------------- VMC-WIDE COUNTERS ----------------
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|5                             
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|3                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|2                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|1                             

