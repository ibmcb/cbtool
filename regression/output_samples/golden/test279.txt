Background operation execution success. Operation "vm-capture" will be processed asynchronously, through the command "/home/msilva/cloudbench/lib/operations//../../cbact --procid=TEST_msilva --osp=port:6379,experiment_inst:TEST_msilva,host:172.16.0.254,pid:TEST_msilva,dbid:10,timout:70,protocol:TCP,kind:redis,username:msilva --msp=port:27017,timeout:240,host:172.16.0.254,pid:TEST_msilva,database:metrics,protocol:TCP,kind:mongodb,username:msilva --oop=TESTCLOUD,vm_41,none --operation=vm-capture --cn=TESTCLOUD --uuid=A244C088-5690-5A3A-8FD5-466C1B66E12C --daemon". The process id is 9274.

Going to unconditionally wait for 25s (25 seconds). The command line interface will be blocked during the waiting.
Waited 10 seconds... (40.0%)
Waited 20 seconds... (80.0%)
Waited 25 seconds... (100.0%)
Waited for 25 seconds.
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_1      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |207.118.209.20  |1372701626     |none                                    |9E9868CD-7406-58B8-A3EA-E5AC86678BE8
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |145.30.115.73   |1372701642     |none                                    |0DDAA48E-8C7B-516D-9320-C823FB84989D
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |183.31.82.55    |1372701680     |none                                    |88BD1BD4-8E44-581E-8858-F7B071ADA048
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |130.243.129.99  |1372701699     |none                                    |22A8D4B3-C2D4-5433-8059-852E6D11B7EA

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |uuid
|vm_26         |db2                 |copper32    |234.36.174.251  |simhosta2                  |SUT                 |ai_1    |none       |712DDE43-5DE3-5510-BF95-CD5BA2F559AE
|vm_28         |client_daytrader    |nano32      |207.118.209.20  |simhostd4                  |LG                  |ai_1    |none       |391B5B6F-F63E-57CC-8217-346BFA00E60D
|vm_33         |client_daytrader    |nano32      |145.30.115.73   |simhostd0                  |LG                  |ai_3    |none       |E4B6738A-D2C7-5339-837B-A8448A06F1B1
|vm_35         |db2                 |copper32    |7.58.141.46     |simhostc4                  |SUT                 |ai_3    |none       |64C4C545-0B6F-565E-9511-3DAAD7E8A2B4
|vm_34         |was                 |iron32      |128.67.222.33   |simhosta2                  |SUT                 |ai_3    |none       |228A2675-52B3-556C-A927-93CAC163C469
|vm_37         |was                 |iron32      |184.196.201.117 |simhosta2                  |SUT                 |ai_4    |none       |6A633C4E-7999-5F41-941E-37B19F48870F
|vm_36         |client_daytrader    |nano32      |183.31.82.55    |simhostd0                  |LG                  |ai_4    |none       |3221F138-2036-5E98-85A6-67541540AD7F
|vm_38         |db2                 |copper32    |153.69.233.76   |simhostb0                  |SUT                 |ai_4    |none       |40FC0736-DF58-5EB7-B37A-D40614D8DCB1
|vm_40         |netclient           |platinum64  |130.243.129.99  |simhostb0                  |SUT                 |ai_5    |none       |857DE764-E6DB-54F8-837F-C8F0F5199F2B
|vm_39         |netserver           |iron32      |145.219.143.248 |simhostb3                  |SUT                 |ai_5    |none       |F9709FFF-07C5-53A1-90E6-C163F31DAA1B
|vm_43         |was                 |iron32      |29.101.197.247  |simhostc3                  |SUT                 |ai_1    |none       |CA0190E3-F9B3-58C7-9C7C-BA349022D969

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |40.73M                        
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |11740532                      
|Total Connections Received                           |6492722                       
|Total Commands Processed                             |92761350                      
|Number of Keys                                       |513                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.0.6                         
|Storage Size                                         |13966491648.0                 
|Data Size                                            |12133993420.0                 
|Index Size                                           |502881232                     
|Average Object Size                                  |6345.87573813                 
|Collections                                          |62                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |11                            
|SVMs                                                 |0                             
|AIs                                                  |4                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |11                            
|VMs ARRIVED                                          |43                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |32                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |4                             
|AIs ARRIVED                                          |7                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |3                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |176                           

 ---------------- VMC-WIDE COUNTERS ----------------
|CB27A579-597C-52AC-B148-644E81A7D93C (simzone_d) VM RESERVATIONS|3                             
|D7D3EB8C-8813-524D-9C2D-7FD7626D4BCC (simzone_b) VM RESERVATIONS|3                             
|97FF48D4-05C4-54D7-B1D8-828EED2A2B20 (simzone_a) VM RESERVATIONS|3                             
|A708E2EA-221A-57ED-BAE8-15077C3E391A (simzone_c) VM RESERVATIONS|2                             

